sap-extensions/extensions.json

```diff
@@ -39,7 +39,29 @@
               "values": {
                 "type": "array",
                 "items": {
-                  "oneOf": [{ "type": "string" }, { "type": "object" }]
+                  "oneOf": [
+                    { "type": "string" },
+                    {
+                      "type": "object",
+                      "additionalProperties": false,
+                      "required": ["text", "format"],
+                      "properties": {
+                        "text": {
+                          "type": "string",
+                          "description": "The actual value for a given extension."
+                        },
+                        "key": {
+                          "type": "string",
+                          "description": "The value key."
+                        },
```

Comment:
@pavelkornev : After our last meeting I think we're now rather clear that we'll not need the "key" here. But we should definately merge the rest of this PR to have a better documentation of the current interface.